charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -75,7 +75,7 @@ spec:
             runAsGroup: 0
           volumeMounts:
             - name: alluxio-fuse-mount
-              mountPath: {{ .Values.fuse.mountPath | dir }}
+              mountPath: {{ .Values.fuse.hostMountPath | dir }}
```

Comment:
In my opinion, I don't think this needs to be changed. We already used the `mountPath | dir` helm function to avoid directly mount the `alluxio-fuse` directory on host.

Reply:
That's the best solution!

Reply:
Yep, i make a mistake. Thanks @TrafalgarZZZ @zwwhdls , i will close this pr.